TestContainer/test/testReleases.js

```diff
@@ -214,3 +218,75 @@ describe("Releases GET", () => {
       });
   });
 });
+
+describe("Releases DELETE", () => {
+  before(() => init());
+  // failure case: Tries to delete without logging in
+  it("DELETE | Should return a 401 unauth", done => {
+    chai
+      .request(process.env.APP_URL)
+      .delete(addressDelete)
+      .end(err => {
+        expect(err.should.have.status(401));
+        done();
+      });
+  });
+
+  // failure case: Tries to delete a non-existant release
+  it("DELETE | Should return a 400 Bad Request", done => {
+    chai
+      .request(process.env.APP_URL)
+      .delete(addressDeleteFail)
+      .set("Content-Type", "application/json")
+      .set("Authorization", "Bearer " + accessToken)
+      .end(err => {
+        expect(err.should.have.status(400));
+        done();
+      });
+  });
+
+  // failure case: Tries to delete a non-existant release without logging in
+  it("DELETE | Should return a 401 unath", done => {
+    chai
+      .request(process.env.APP_URL)
+      .delete(addressDeleteFail)
+      .end(err => {
+        expect(err.should.have.status(401));
+        done();
+      });
+  });
+
+  // success case: deletes a release
+  it("DELETE | Should return a ok 200, and a copy of the deleted release", done => {
+    chai
+      .request(process.env.APP_URL)
+      .delete(addressDelete)
+      .set("Content-Type", "application/json")
+      .set("Authorization", "Bearer " + accessToken)
+      .end((err, res) => {
+        if (err) {
+          done(err.response.text);
+        }
+        res.should.have.status(200);
+        expect(res.body.id).to.equal(103);
+        expect(res.body.title).to.equal("2012");
+        expect(res.body.isPublic).to.equal(false);
+        expect(res.body.productVersion).to.be.not.empty;
+        expect(res.body.releaseNotes).to.be.a("array").that.is.empty;
+        done();
+      });
+  });
+
+  // failure case: Tries to delete a already deleted release
+  it("DELETE | Should return 400 and a error message", done => {
```

Comment:
"and a error message" du sjekk kun statuskode, som eg mein e nok, men ikke jug.

Reply:
ok, skal korrigere kommentaren. Bytter også om på hva som er i describe og hva som er i kommentarer.